app/src/main/java/app/nexd/android/ui/seeker/create/SeekerCreateRequestFragment.kt

```diff
@@ -22,64 +23,49 @@ class SeekerCreateRequestFragment : Fragment() {
+    private lateinit var binding: FragmentSeekerCreateRequestBinding
+
+    private val adapter = MultiViewAdapter()
 
     override fun onCreateView(
         inflater: LayoutInflater, container: ViewGroup?,
         savedInstanceState: Bundle?
     ): View? {
-        return inflater.inflate(R.layout.fragment_seeker_create_request, container, false)
+        binding = FragmentSeekerCreateRequestBinding.inflate(inflater, container, false)
+        binding.viewModel = vm
+        binding.lifecycleOwner = viewLifecycleOwner
+        return binding.root
     }
 
-    override fun onActivityCreated(savedInstanceState: Bundle?) {
-        super.onActivityCreated(savedInstanceState)
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
 
         recyclerView_requests.layoutManager = LinearLayoutManager(context)
-
-        adapter = MultiViewAdapter()
         recyclerView_requests.adapter = adapter
 
         adapter.registerItemBinders(HelpRequestCreateArticleBinder())
 
-        vm.getCurrentUser().observe(viewLifecycleOwner, Observer { currentUser ->
-
-            vm.getArticles().observe(viewLifecycleOwner, Observer { articles ->
-                adapter.removeAllSections()
-
-                val articlesSection = ListSection<HelpRequestCreateArticleBinder.ArticleInput>()
-                val articlesInput = articles.map { HelpRequestCreateArticleBinder.ArticleInput(it) }
-                articlesSection.addAll(articlesInput)
-
-                adapter.addSection(articlesSection)
-
-                button_accept.setOnClickListener {
-                    val request = HelpRequestCreateDto()
-                        .articles(articlesInput
-                            .filter { it.amount > 0 }
-                            .map {
-                                CreateHelpRequestArticleDto()
-                                    .articleCount(it.amount)
-                                    .articleId(it.article.id)
+        vm.articles.observe(viewLifecycleOwner, Observer {
+            adapter.removeAllSections()
+            val articlesSection = ListSection<HelpRequestCreateArticleBinder.ArticleInput>()
+            articlesSection.addAll(it)
+            adapter.addSection(articlesSection)
+        })
 
-                            })
-                        .street(currentUser.street)
-                        .number(currentUser.number)
-                        .zipCode(currentUser.zipCode)
-                        .city(currentUser.city)
-                        .phoneNumber(currentUser.phoneNumber)
-                        .additionalRequest(textView_additionalRequest.text.toString())
+        button_abort.setOnClickListener {
+            findNavController().navigateUp()
+        }
 
-                    vm.sendRequest(request)
+        vm.progress.observe(viewLifecycleOwner, Observer { progress ->
+            when (progress) {
+                is Idle -> {
```

Comment:
Do we need this state or can we maybe remove it ?

Reply:
if we want to disable the buttons during loading later on, the Idle state would be important to enable them